Shoko.Server/API/v2/Modules/PlexWebhook.cs

```diff
@@ -179,10 +184,31 @@ private void Scrobble(PlexEvent data, SVR_JMMUser user)
                 break;
         }
 
+        var key = metadata.Key.Split("/").LastOrDefault();  // '/library/metadata/{key}'
+
+        var plexEpisode = (SVR_Episode)GetPlexEpisodeData(key, user);
+        var episode = plexEpisode?.AnimeEpisode;
+
         if (episodeType != EpisodeType.Episode ||
             metadata.Index == 0) //metadata.index = 0 when it's something else.
         {
-            return (null, anime); //right now no clean way to detect the episode. I could do by title.
+            if (episode == null)
+            {
+                _logger.LogInformation(
+                    $"Failed to get anime episode from plex using key {metadata.Key}.");
```

Comment:
```suggestion
                _logger.LogInformation($"Failed to get anime episode from plex using key {metadata.Key}.");
```

Code style - there's no real need to have it on multiple lines here.